tests/test_mux.py

```diff
@@ -582,6 +584,7 @@ def create_and_test_NH_routes(self, appdb, asicdb, dvs, dvs_route, mac):
 
         self.add_neighbor(dvs, neigh_ip, mac)
         self.add_neighbor(dvs, neigh_ipv6, mac)
+        time.sleep(1)
```

Comment:
@Ndancejic can we use a more robust synchronization approach here? Let's say waiting for added/removed entries in APP/ASIC DB?

Reply:
updated it so we wait for db entries instead of sleeping

Reply:
@Ndancejic thanks! Looks like the polling interval is too small, so PR keeps failing. Can you please have a look?